src/server/ApiRoutes/Hold.js

```diff
@@ -118,13 +118,15 @@ function getDeliveryLocations(barcode, patronId, accessToken, cb, errorCb) {
     }
   )
   .then(barcodeAPIresponse => {
-    const deliveryLocationWithAddress = mapLocationDetails(
-      barcodeAPIresponse.data.itemListElement[0].deliveryLocation
-    );
+    const eddRequestable = (barcodeAPIresponse.data.itemListElement[0].eddRequestable) ?
```

Comment:
Curious, does the barcode endpoint always return an array with one element? If it returns more than one, what does it mean?